conf/messages.cy

```diff
@@ -154,4 +154,90 @@ adviceSuccess.p2.4 = nodi bod y cofnod yn anghywir, ac yn esbonio pam
+goodsDescription.inputHeading = Beth yw’r disgrifiad o’r nwyddau?
+goodsDescription.hint = Gall hyn fod hyd at 512 o gymeriadau.
+goodsDescription.checkYourAnswersLabel = Disgrifiad o’r nwyddau
+
+commodityCode.title = Cod nwyddau
+commodityCode.h1 = Cod nwyddau
+commodityCode.p1 = Mae codau nwyddau yn gyfeirnodau sy’n cael eu cydnabod yn rhyngwladol. Mae cod nwyddau yn disgrifio cynnyrch penodol wrth fewnforio neu allforio nwyddau. Byddwch yn defnyddio’r cod hwn ar unrhyw ddatganiadau tollau.
+commodityCode.p2 = Gallwch ychwanegu codau nwyddau i Broffil Nwyddau Masnachwr mewn 6, 8 neu 10 digid.
+commodityCode.p3 = Os nad ydych yn siŵr beth yw cod nwyddau eich nwyddau, gallwch ddefnyddio
+commodityCode.p3.linkText = Offeryn Tariff Ar-lein Gogledd Iwerddon
+commodityCode.heading = Beth yw’r cod nwyddau ar gyfer eich nwyddau?
+commodityCode.hint = Gallwch ddefnyddio 6, 8 neu 10 digid, er enghraifft, 654321, 87654321, 9876543211
+
+cyaCreateRecord.title = Gwiriwch eich atebion
+cyaCreateRecord.h1 = Gwiriwch eich atebion
+cyaCreateRecord.h2 = Gwiriwch fod y manylion yn y cofnod hwn yn gywir.
+cyaCreateRecord.warning = Rhybudd
+cyaCreateRecord.warningText = Rydych yn agored am y wybodaeth am unrhyw symudiadau a grëir gennych chi, neu ar eich rhan gan ddefnyddio eich cofnodion Proffil Nwyddau Masnachwr.
+
+
+createRecordSuccess.title = Rydych wedi creu cofnod nwyddau
+createRecordSuccess.h1 = Rydych wedi creu cofnod nwyddau
+createRecordSuccess.h2 = Yr hyn sy’n digwydd nesaf
+createRecordSuccess.p2 = Mae dal angen i chi categoreiddio’r nwyddau cyn i chi ei defnyddio ar symudiadau.
+createRecordSuccess.p3 = Nid oes rhaid i chi gategoreiddio’r cofnod hwn nawr. Gallwch adolygu’r cofnod o’ch rhestro nwyddau a dechrau categoreiddio yn ddiweddarach.
+createRecordSuccess.p4.linkText = Categoreiddio nwyddau nawr
+createRecordSuccess.p5.linkText = Ewch i’r hafan
+createRecordSuccess.h2.1 = Rhowch eich barn
+createRecordSuccess.p6 = Mae hwn yn wasanaeth newydd ac rydym eisiau gwybod beth yw’ch barn amdano.
+createRecordSuccess.p7.linkText = Anfonwch eich adborth atom.
+createRecordSuccess.p7 = Bydd eich meddyliau a’ch syniadau’n ein helpu i wneud gwelliannau.
+
+traderReference.error.required = Nodwch y cyfeirnod masnachwr
+traderReference.error.traderRefNotUnique = Mae’r cyfeirnod masnachwr hwn eisoes yn eich Proffil Nwyddau Masnachwyr. Nodwch gyfeirnod masnachwr unigryw.
+useTraderReference.error.required = Cadarnhewch a ydych am ddefnyddio’ch cyfeirnod masnachwr fel disgrifiad y nwyddau
+goodsDescription.error.required = Nodwch ddisgrifiad o’r nwyddau
+goodsDescription.error.length = Mae’n rhaid i ddisgrifiad y nwyddau fod yn 512 o gymeriadau neu lai
+countryOfOrigin.error.required = Nodwch y wlad wreiddiol
+commodityCode.error.invalidFormat = Nodwch god nwyddau yn y fformat cywir
+commodityCode.error.invalid = Nodwch god nwyddau go iawn
+
```

Comment:
site.back needs a translation too (the back button) there needs to be a ticket for this if there isn't already or it needs to be done in this one

Reply:
also site.save.continue